topics/ecology/tutorials/Ecoregionalization_tutorial/tutorial.md

```diff
@@ -0,0 +1,352 @@
+
+This is an example of environmental file input in this workflow : 
+
++------+------+---------+------+------+
+| long | lat  |  Carbo  | Grav | ...  |
++------+------+---------+------+------+
+|139.22|-65.57|   0.88  |28.59 | ...  |
++------+------+---------+------+------+
+|139.22|-65.57|   0.88  |28.61 | ...  |
++------+------+---------+------+------+
+| ...  | ...  |   ...   | ...  | ...  |
++------+------+---------+------+------+
+
+## Occurrence data
+
+The second data file you will need to run this workflow is occurrences data file. As defined above, occurrences data showing the presence of a species at a particular location. This data file, also need to be in tabular format (.tsv) and need to be construct as following : 
+
+- columns latitude and longitude
+
+- one column per taxon where each box corresponding to a geographical point is marked 1 if the taxon is present or 0 if the taxon is absent 
+
+- the environmental characteristics corresponding to the geographical coordinates. 
+
+This is an example of occurrence data file input in this workflow :
+
++---------+----------+-----------+------------------------+-----------+-----+------+--------------+-----+
+| station |   lat    |   long    |Acanthorhabdus_fragilis | Acarnidae | ... | Grav |  Maxbearing  | ... |
++---------+----------+-----------+------------------------+-----------+-----+------+--------------+-----+
+|1        |-65,9     |142,3      |1                       |0          | ... |28.59 |3.67          | ... |
++---------+----------+-----------+------------------------+-----------+-----+------+--------------+-----+
+|10       |-66,3     |141,3      |0                       |1          | ... |28.61 |3.64          | ... |
++---------+----------+-----------+------------------------+-----------+-----+------+--------------+-----+
+|   ...   |   ...    |   ...     |...                     |    ...    | ... | ...  |...           | ... |
++---------+----------+-----------+------------------------+-----------+-----+------+--------------+-----+
+
+## Data upload
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial give it a name (example: “Ecoregionalization workflow”) for you to find it again later if needed.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+>    {% snippet faqs/galaxy/histories_rename.md %}
+>
+> 2. Import the environement file and occurrence file(s) from your local environement.
+>   
+>    {% snippet faqs/galaxy/datasets_import_from_remote_files.md %}
+>
+> 3. Rename the datasets if needed. 
+>
+> {% snippet faqs/galaxy/datasets_rename.md %}
+>
+> 4. Check that the datatype is tabular.
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="tabular" %}
+>
+> 5. Add to each occurrence data file (if you have multiple files) a tag to avoid mixing them.
+>
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+{: .hands_on}
+
+# First step with **BRT tool prediction**
```

Comment:
```suggestion
# Predicting taxa distribution with BRT
```

Generally the GTN prefers this sort of section heading, rather than "№ step with Tool X", saying what scientific thing will be done in this section